var/spack/repos/builtin/packages/clingo/package.py

```diff
@@ -64,6 +64,8 @@ class Clingo(CMakePackage):
         depends_on("py-cffi", type=("build", "run"), when="@5.5.0: platform=cray")
 
     patch("python38.patch", when="@5.3:5.4.0")
+    patch("size-t.patch", when="platform=windows")
```

Comment:
Is this compiler dependent?

Reply:
`ssize_t` is defined in `<unistd.h>`, so this should be compiler dependent.
Should I change it to something like `when="%msvc"`?

Reply:
That would be preferable yes. Leaving the conditional so open ended may result in inscrutable errors down the line.

Reply:
Modified in the new commit `408f415`.